charts/my-bloody-jenkins/templates/ingress.yaml

```diff
@@ -35,9 +41,12 @@ spec:
   - http:
       paths:
         - path: {{ .Values.ingress.path }}
+          pathType: {{ .Values.ingress.pathType }}
           backend:
-            serviceName: {{ $fullName }}
-            servicePort: http
+            service: 
```

Comment:
This change won't work with `extensions/v1beta1`
You will need to wrap it with if/else so we won't break older versions